scripts/zephyr_module.py

```diff
@@ -130,7 +130,7 @@ def main():
         out, err = p.communicate()
         if p.returncode == 0:
             projects = out.decode(sys.getdefaultencoding()).splitlines()
-        elif re.match(r'Error: .* is not in a west installation\..*',
+        elif re.match(r'FATAL ERROR: no west installation found from .*',
```

Comment:
Thanks for this -- can you match against both REs to keep it working with both versions using a `|`?

Reply:
I could do that, sure, but I will argue it's not needed. If you have an older West version you will fail earlier with a message that 0.6.0 is required anyway - so it might be a bit confusing.

But if you reconfirm you think it's a good idea, I will add it